_tour/implicit-conversions.md

```diff
@@ -23,21 +23,22 @@ Implicit conversions are applied in two situations:
 In the first case, a conversion `c` is searched for which is applicable to `e` and whose result type conforms to `T`.
 In the second case, a conversion `c` is searched for which is applicable to `e` and whose result contains a member named `m`.
 
-The following operation on the two lists xs and ys of type `List[Int]` is legal:
+If an implicit method `List[A] => Ordered[List[A]]` like the one below
 
 ```
-xs <= ys
+implicit def list2ordered[A](x: List[A])
+    (implicit elem2ordered: A => Ordered[A]): Ordered[List[A]] =
+  new Ordered[List[A]] { 
+    //replace with a more useful implementation
+    def compare(that: List[A]): Int = 1
+  }
 ```
 
-assuming the implicit methods `list2ordered` and `int2ordered` defined below are in scope:
+is in scope, as well as an implicit method `Int => Ordered[Int]`, one of which is provided automatically
+through `scala.Predef.intWrapper`, the following operation on the two lists of type `List[Int]` is legal:
```

Comment:
Probably better to explain _which_ is provided by `intWrapper`.

Reply:
Ha, I meant that `intWrapper` provides "some" `Int => Ordered[Int]` out of many possible implementations, but indeed this reads totally ambiguous.